tests/cisco_xr/show_interfaces_summary/cisco_xr_show_interfaces_summary.yml

```diff
@@ -0,0 +1,2 @@
+---
+parsed_sample: []
```

Comment:
Making the change above on the index file will fix this and give you the parsed data from the raw file.

Reply:
I did the change on index file and also the change with the new structure:

`cisco_xr_show_redundancy_summary.textfsm, .*, cisco_xr, sh[[ow]] redun[[dancy]] summ[[ary]]
`

```
Value INTF_TYPE ((\S+\s+\S+)|(\S+))
Value TOTAL (\d+)
Value UP (\d+)
Value DOWN (\d+)
Value ADMIN_DOWN (\d+)

Start
  ^Interface\s+Type\s+Total\s+UP\s+\Down\s+Admin\s+Down\s*$$
  ^----
  ^${ALL_TYPES}+\s+${TOTAL}\s+${UP}\s+${DOWN}\s+${ADMIN_DOWN} -> Record
  ^\s*$$
  ^. -> Error
```
But the result was the same:

`
---
parsed_sample: []

`

Reply:
I did the change on index file and also the change with the new structure:
`cisco_xr_show_interfaces_summary.textfsm, .*, cisco_xr, sh[[ow]] int[[erfaces]] summ[[ary]]`

`Value INTF_TYPE ((\S+\s+\S+)|(\S+))
Value TOTAL (\d+)
Value UP (\d+)
Value DOWN (\d+)
Value ADMIN_DOWN (\d+)

Start
  ^Interface\s+Type\s+Total\s+UP\s+\Down\s+Admin\s+Down\s*$$
  ^----
  ^${ALL_TYPES}+\s+${TOTAL}\s+${UP}\s+${DOWN}\s+${ADMIN_DOWN} -> Record
  ^\s*$$
  ^. -> Error`

At the end, both templates (my version and the proposed version changing `^${ALL_TYPES}` to `^${INTF_TYPE}`, create the same yml result.

So, what template version do I have to apply?

Reply:
`^${ALL_TYPES}+\s+${TOTAL}\s+${UP}\s+${DOWN}\s+${ADMIN_DOWN} -> Record`

This was a typo, and should have been `^${INTF_TYPE}+\s+${TOTAL}\s+${UP}\s+${DOWN}\s+${ADMIN_DOWN} -> Record`